src/aspire/utils/importlib_legacy.py

```diff
@@ -0,0 +1,58 @@
+import os
+import pathlib
+import types
+from typing import Any, ContextManager, TextIO, Union
+
+import importlib_resources
+
+Package = Union[types.ModuleType, str]
+Resource = str
+
+# The purpose of this file is to suppress deprecation warnings arising
+# from using deprecated legacy methods in the importlib_resources library.
+# The methods below are still supported in importlib_resources._legacy
+# but they raise warnings alerting that the new API should be used.
+
+# We plan to move to importlib.resources, which importlib_resources is a
+# backport of, once we have dropped support for Python 3.6. At that point
+# the package resource management code in the tests will need to be
+# rewritten with importlib.resources. This file will then be deleted.
```

Comment:
This is good, but add a few hyperlinks including where we copied the patches from.  Thanks.

Reply:
Perfect thanks!